docs/building.md

```diff
@@ -0,0 +1,176 @@
+# Cloning and building
+
+Fobnail project consists of 3 main repositories with code:
+
+- [Fobnail firmware](https://github.com/fobnail/fobnail) (Rust)
+- [Platform Owner](https://github.com/fobnail/fobnail-platform-owner) (C)
+- [Attester](https://github.com/fobnail/fobnail-attester) (C)
+
+There is also [Fobnail SDK](https://github.com/fobnail/fobnail-sdk) that
+simplifies building of firmware by using Docker container, but only
+[one script](https://github.com/fobnail/fobnail-sdk/blob/main/run-fobnail-sdk.sh)
+from this repository is actually needed.
+
+Following steps were successfully performed on Ubuntu 22.04.1 LTS.
+
+## Fobnail Token firmware
+
+Start by installing prerequisites and adding current user to `docker` group:
+
+```shell
+sudo apt -y install git docker.io
+sudo usermod -aG docker $USER
+```
+
+Log out of the desktop session then log in again - this is required after making
+the change to user groups. You may confirm it by running `groups`.
```

Comment:
I will move whole part about Fobnail SDK to another file, it will be referenced from multiple places.

Reply:
Does Dasharo support Linuxes other than Ubuntu?

Reply:
@krystian-hebel please link such section for review so I can solve this conversation.

> Does Dasharo support Linuxes other than Ubuntu?

Officially not. What Linux distro in your opinion we should also offcially support?

Reply:
I was just asking because Docker documentation links to Ubuntu instructions only, even though they are mostly identical for all Debian-based distributions.

Anyway, instructions were added to `docs/fobnail-sdk.md` on this branch.